### PR TITLE
Fix interface/qwbackendinterface.h: No such file or directory

### PR DESCRIPTION
**Pull request by zccrs**
When using QWlroots from system by find_package(QWlroots), because the "interface" directory is not install to system, so the project build failed.

### CODE REVIEW

**Comment by wineee** [comment]
Shouldn't we install the `interface` to the system?

**Comment by zccrs** [comment]
> Shouldn't we install the `interface` to the system?

The "qwbackendinterface.h" file will install to system, but don't install the "interface" sub directory.

**Review by wineee — APPROVED** [review]

File: src/types/qwinputdevice.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include "qwkeyboard.h"
 #include "qwpointer.h"
 #include "qwtablet.h"
-#include "qwtabletpad.h""
```
Comment (wineee) [inline src/types/qwinputdevice.cpp:9]: Can we add a github bot to check code format，for example https://github.com/marketplace/codefactor

Comment (zccrs) [reply src/types/qwinputdevice.cpp:9]: Try it.

**Pull request merged**